generic.js

```diff
@@ -2,4 +2,6 @@
 
 if(vehicle === "motorbike") {
   console.log("You should add a couple of wheels")
+} else if(vehicle ===    "car") {
+  console.log('Life is good')
```

Comment:
Codacy found an issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26445)
```suggestion
  console.log("Life is good")
```